Sources/UID2/UID2Manager.swift

```diff
@@ -41,8 +40,20 @@ public final actor UID2Manager {
     /// UID2Client for Network API  requests
     private let uid2Client: UID2Client
     
-    private let timer: RepeatingTimer
+    /// Background Task for Refreshing UID2 Identity
+    private var refreshJob: Task<(), Error>?
+
+    /// Background Task for Checking On UID2 Identity Refresh Token Expiration
+    private var checkRefreshExpiresJob: Task<(), Error>?
+
+    /// Background Task for Checking On UID2 Identity Expiration
+    private var checkIdentityExpiresJob: Task<(), Error>?
+    
+    /// Toggle for Expiration Checks Functionality
+    private let checkExpiration = true
```

Comment:
It doesn't look like this is ever used. The reason I added it on `Android` was a way for my Unit Tests to control whether or not that behaviour was enabled. The async nature of this class made it hard to set up different tests. I therefore had this disabled for unrelated tests and only enabled it when specifically testing for Identity expiration.